### PR TITLE
[NumericInput]: deprecated size 48

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: uui/components/inputs/TextInput.tsx [hunk 1]
```diff
@@ -5,11 +5,18 @@ import { IHasEditMode, EditMode, ControlSize } from '../types';
 import { systemIcons } from '../../icons/icons';
 import css from './TextInput.module.scss';
 
+/**
+ * Represents the default size for a specific feature.
+ */
 const defaultSize = '36';
+/**
+ * Represents the default mode for editing a form.
```
Comment (AlekseyManetov) [inline uui/components/inputs/TextInput.tsx:13]: let's remove it since it's not a type or interface

File: uui/components/widgets/Badge.tsx [hunk 2]
```diff
@@ -17,22 +18,32 @@ const mapSize = {
 export const defaultBadgeSize = '36';
 export const defaultBadgeFill = 'solid';
 
+/** Represents the types of badge fills available. */
 export type BadgeFill = 'solid' | 'outline';
```
Comment (AlekseyManetov) [inline uui/components/widgets/Badge.tsx:22]: i guess we can remove export here

File: uui/components/widgets/Badge.tsx [hunk 3]
```diff
@@ -17,22 +18,32 @@ const mapSize = {
...
+/** Represents the size options for a badge. */
 export type BadgeSize = '18' | '24' | '30' | '36' | '42' | '48';
```
Comment (AlekseyManetov) [inline uui/components/widgets/Badge.tsx:24]: i guess we can remove export here

File: uui/components/widgets/Badge.tsx [hunk 4]
```diff
@@ -17,22 +18,32 @@ const mapSize = {
...
+/** Represents the color of a badge. */
 export type BadgeColor = 'info' | 'success' | 'warning' | 'critical' | 'neutral';
```
Comment (AlekseyManetov) [inline uui/components/widgets/Badge.tsx:26]: i guess we can remove export here

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]
please fix suggestions

File: epam-promo/components/widgets/Badge.tsx [hunk 5]
```diff
@@ -4,11 +4,20 @@ import { EpamPrimaryColor } from '../types';
 import css from './Badge.module.scss';
 
 export type BadgeMods = {
-    /** @default 'blue' */
+    /**
+     * The color of an element.
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/components/widgets/Badge.tsx]: fixed

File: epam-promo/components/widgets/Badge.tsx [hunk 6]
```diff
@@ -4,11 +4,20 @@ import { EpamPrimaryColor } from '../types';
...
+     * @default 'blue'
+     */
     color?: EpamPrimaryColor | 'yellow'| 'orange' | 'fuchsia' | 'purple' | 'violet' | 'cobalt' | 'cyan' | 'mint' | 'white' | 'gray10'| 'gray30'| 'gray60';
-    /** @default 'solid' */
+    /**
+     * The fill property of the Badge component.
+     * @default 'solid'
+     */
     fill?: uui.BadgeMods['fill'] | 'semitransparent';
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/components/widgets/Badge.tsx:16]: fixed

File: epam-promo/components/widgets/Badge.tsx [hunk 7]
```diff
@@ -4,11 +4,20 @@ import { EpamPrimaryColor } from '../types';
...
-    /** @default '36' */
+    /**
+     * Represents the size of a badge.
+     * @default 36
+     */
     size?: uui.BadgeMods['size'];
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/components/widgets/Badge.tsx:21]: fixed

File: epam-promo/components/widgets/Badge.tsx [hunk 8]
```diff
@@ -4,11 +4,20 @@ import { EpamPrimaryColor } from '../types';
 import css from './Badge.module.scss';
 
 export type BadgeMods = {
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/components/widgets/Badge.tsx]: fixed

File: epam-promo/components/buttons/IconButton.tsx [hunk 9]
```diff
@@ -2,15 +2,20 @@ import { createSkinComponent } from '@epam/uui-core';
 import { EpamPrimaryColor } from '../types';
 import * as uui from '@epam/uui';
 
+/** Represents a color for an icon. */
 export type IconColor = EpamPrimaryColor | 'gray30' | 'gray50' | 'gray60';
```
[the remark this answers is not in the document]
Comment (vik753) [reply epam-promo/components/buttons/IconButton.tsx]: fixed

File: uui/components/inputs/TextInput.tsx (hunk 1 above)
Comment (vik753) [reply uui/components/inputs/TextInput.tsx]: fixed

File: uui/components/widgets/Badge.tsx (hunk 2 above)
Comment (vik753) [reply uui/components/widgets/Badge.tsx]: fixed

File: uui/components/widgets/Badge.tsx (hunk 3 above)
Comment (vik753) [reply uui/components/widgets/Badge.tsx]: fixed

**Pull request merged**